src/txn/txn.c

```diff
@@ -1019,10 +1019,8 @@ __wt_txn_commit(WT_SESSION_IMPL *session, const char *cfg[])
                 /*
                  * Writes to the lookaside file can be evicted as soon as they commit.
                  */
-                if (conn->cache->las_fileid != 0 && fileid == conn->cache->las_fileid) {
-                    upd->txnid = WT_TXN_NONE;
+                if (conn->cache->las_fileid != 0 && fileid == conn->cache->las_fileid)
```

Comment:
Enhance the comment, to say that we use different timestamps for different entries and we don't want to use __wt_txn_op_set_timestamp which will overwrite the timestamps of all the operations.

Reply:
Updated comment to explain the reason.